src/index.js

```diff
@@ -10,14 +12,41 @@ import rootReducer from "./reducers/root";
 import * as serviceWorker from "./serviceWorker";
 
 import { loginWithBakery } from "./juju";
+import jujuReducers from "./juju/reducers";
+import { fetchModelList } from "./juju/actions";
+import { actionsList } from "./reducers/actions";
 
-loginWithBakery();
+const reduxStore = createStore(
+  combineReducers({
+    root: rootReducer,
+    juju: jujuReducers
+  }),
+  composeWithDevTools(applyMiddleware(thunk))
+);
+
+// eslint-disable-next-line no-shadow
```

Comment:
What's the reason for this exception?

Reply:
Ahh, because I have a variable called `reduxStore` in the `connectAndListModels` and one in the app scope. It's not really a shadow because the app scoped one is defined by const and cannot be overridden without an error so I added the exception.